pom.xml

```diff
@@ -128,7 +128,10 @@
     <ning.async.version>1.8.12</ning.async.version>
     <snappy.version>0.3</snappy.version>
     <sentry.version>6.0.0</sentry.version>
-    <horizon.version>0.0.20</horizon.version>
+    <horizon.version>0.0.25</horizon.version>
```

Comment:
Any reason not to go to `0.1.1`?

Reply:
Oh I just didn't realize it was out. I'll change that.